examples/sparse_blas/compile_time_dispatching/CMakeLists.txt

```diff
@@ -18,27 +18,24 @@
 #===============================================================================
 
 #Build object from all sources
-set(SPARSE_BLAS_BACKENDS "")
-
-if(ENABLE_MKLCPU_BACKEND)
-  list(APPEND SPARSE_BLAS_BACKENDS "mklcpu")
+set(SPARSE_CT_SOURCES "")
+if(ENABLE_MKLCPU_BACKEND AND ENABLE_CUSPARSE_BACKEND)
```

Comment:
Why is `ENABLE_MKLGPU_BACKEND` missing here? Or maybe should it be `(ENABLE_MKLCPU_BACKEND AND ENABLE_MKLGPU_BACKEND) OR (ENABLE_MKLCPU_BACKEND AND ENABLE_CUSPARSE_BACKEND)`?

Reply:
This is chosen as an example, the same backends are used for the other compile time example in the other domains, see blas for instance: https://github.com/oneapi-src/oneMKL/blob/develop/examples/blas/compile_time_dispatching/level3/CMakeLists.txt#L22